dev/ScrollViewer/APITests/ScrollViewerTests.cs

```diff
@@ -392,33 +625,76 @@ private void WaitForEvent(string logComment, EventWaitHandle eventWaitHandle)
                 throw new Exception("Timeout expiration in WaitForEvent.");
             }
         }
+    }
 
-        private void MUXControlsTestHooks_LoggingMessage(object sender, MUXControlsTestHooksLoggingMessageEventArgs args)
+    // Custom ScrollViewer that records its visual state changes.
+    public class ScrollViewerVisualStateCounts
+    {
+        public uint NoIndicatorStateCount
         {
-            string msg = args.Message.Substring(0, args.Message.Length - 1);
-            string senderName = string.Empty;
+            get;
```

Comment:
If you're not doing anything with the get/sets you might as well make these fields. Ulterior motive: You could then more easily clean up MUXControlsTestHooks_LoggingMessageForVisualStateChange to a single worker function (takes string and reference to one of these fields).

Reply:
I might do that in a future check-in. Getting it in as is for now.